### DISCUSSION TITLE
is it possible to fix first column in resourceTimeGridWeek and resourceTimeGridDay view and make resources scrollable?

### DISCUSSION THREAD

**Discussion by worxflowadmin** [vkurko/calendar#209] [Q&A] [answered]
Hello guys,

is it possible to fix first column in resourceTimeGridWeek and resourceTimeGridDay view and make resources scrollable?
So that time column which is first column and can be fixed and Resources section on right can be scrollable horizontally left and right?
I have attached image below.

![Firefox_Screenshot_2023-11-27T14-00-08 392Z](https://github.com/vkurko/calendar/assets/151848885/a9a55bda-73eb-45e4-a0ac-4d7848ecde41)

Please let me know if that is possible somehow.

Thank you.

**Comment by vkurko** [comment, marked as answer]
There is no out-of-the-box solution to get this result. Maybe someone could do it with additional CSS styles, but I'm not aware of it.
